src/js/component.js

```diff
@@ -130,53 +167,62 @@ class Component {
   }
 
   /**
-   * Return the component's player
+   * Return the {@link Player} that the `Component` has attached to.
    *
    * @return {Player}
-   * @method player
+   *         The player that this `Component` has attached to.
    */
   player() {
     return this.player_;
   }
 
   /**
-   * Deep merge of options objects
-   * Whenever a property is an object on both options objects
-   * the two properties will be merged using mergeOptions.
+   * Deep merge of options objects with new options.
+   * > Note: When a property of is an object on both `obj` and `options`.
```

Comment:
"When a property of is an object" is weird.
Maybe something like:
When both `obj` and `options` contain a property whose value is an object, the two properties get merged using mergeOptions